packages/plugin/src/theia.d.ts

```diff
@@ -5377,6 +5377,17 @@ export module '@theia/plugin' {
          */
         password: boolean;
 
+        /**
+         * Selection range in the input value. Defined as tuple of two number where the
+         * first is the inclusive start index and the second the exclusive end index. When `undefined` the whole
+         * pre-filled value will be selected, when empty (start equals end) only the cursor will be set,
+         * otherwise the defined range will be selected.
+         *
+         * This property does not get updated when the user types or makes a selection,
+         * but it can be updated by the extension.
+         */
+        valueSelection: readonly [number, number];
```

Comment:
This doesn't seem to align with the [vscode definition of `valueSelection`](https://github.com/microsoft/vscode/blob/a606c86868801bdd6a3f4ddd3ba6253d068d0ed5/src/vscode-dts/vscode.d.ts#L11544), which has an additional ` | undefined`.